src/renderer/src/stories/FileSystemSelector.js

```diff
@@ -125,6 +127,11 @@ export class FilesystemSelector extends LitElement {
 
     #useElectronDialog = async (type) => {
         const options = { ...this.dialogOptions };
+
+        if (!options.filters && this.accept) {
+            options.filters = [{ name: "Selected Files", extensions: this.accept }];
```

Comment:
Maybe just need to add another element here?

```suggestion
            options.filters = [{ name: "Selected Files", extensions: this.accept }, {name: "All Files", extensions: ["*"]}];
```

Reply:
Yep this allows for changing the filter. Thanks!